.storybook/manager.js

```diff
@@ -10,3 +10,7 @@ import theme from './theme';
 addons.setConfig({
   theme,
 });
+
+// Set the Google Analytics tracking ID if the `master` branch is deployed.
+window.STORYBOOK_GA_ID =
+  process.env.STORYBOOK_BRANCH === 'master' && 'UA-80770450-3';
```

Comment:
Not sure if there is much you can do about this now, but should the GA ID be provided as an environment variable as well to keep it hidden from source control?

Reply:
😆  Yeah, it's a bit late - the rationale at the time was that it would still be accessible either by viewing the website's source or by logging `window.STORYBOOK_GA_ID` to the console.

Granted it takes a bit more work than just browsing source code, but I do know there are spam filters we can leverage if the need arises.

Reply:
If there is a way to regenerate that ID and store it in an environment variable, I would do so.